.pylintrc

```diff
@@ -11,6 +11,7 @@ disable=cyclic-import,
         star-args,
         too-few-public-methods,
         too-many-instance-attributes,
+        raise-missing-from,
```

Comment:
The version of Pylint in tox.ini is not pinned, i.e. we get the latest version automatically each time we trigger a build.  I didn't think this new "raise-missing-from" rule would significantly improve the code readability and updating the code to comply with the recommendations from https://www.python.org/dev/peps/pep-3134/#explicit-exception-chaining would distract from the primary intent of this PR.  So I have disabled it for now.

Reply:
An alternative is to do in-line disables on the individual lines, if you think the global disable is too much

Reply:
You can disable per file which I'd be OK with in this case.  Or I'm happy to leave it disabled globally.  But I think disabling on each of the 4 affected lines would add just as much noise to the PR as actually addressing the recommendation, i.e.


```
        except hvac.exceptions.VaultError as err:
            msg = err.args[0]
            raise OperationalError(msg)  # pylint: disable=raise-missing-from
```
versus
```
        except hvac.exceptions.VaultError as err:
            msg = err.args[0]
            raise OperationalError(msg) from err
```

It theoretically should make tracebacks more comprehensive, but I've never actually come across a case where I wished that the traceback included better exception chaining.

Reply:
Global is good